### PR TITLE
SCRP-1346: use vars.GITHUB_ACTOR and secrets.GITHUB_TOKEN

### PR DESCRIPTION
**Pull request by kenzhang-ces**
Projects that may use this workflow:

- ms-pos-refund-service
- ms-nop-api
- ms-login-test
- ms-payments-approval
- ms-pos-api
- ms-braze-adapter-api

 build.yml test here: https://github.com/coexservices/ms-pos-refund-service/actions/runs/4351879563 
validate-pr.yml test here: https://github.com/coexservices/ms-pos-refund-service/actions/runs/4351877586

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -58,8 +58,8 @@ jobs:
       run: |
         mkdir -p $HOME/.gradle
         echo "allprojects { repositories { mavenCentral(); maven { name \"GitHubPackages\"; url uri(\"https://maven.pkg.github.com/coexservices/*\"); credentials { username githubRepoActor; password githubRepoToken } } } }" > ~/.gradle/init.gradle
-        echo "githubRepoActor=${{ secrets.github_repo_actor }}" > ~/.gradle/gradle.properties
-        echo "githubRepoToken=${{ secrets.github_repo_token }}" >> ~/.gradle/gradle.properties
+        echo "githubRepoActor=${{ vars.GITHUB_ACTOR }}" > ~/.gradle/gradle.properties
+        echo "githubRepoToken=${{ secrets.GITHUB_TOKEN }}" >> ~/.gradle/gradle.properties
```
Comment (jsoetanto-ces) [inline .github/workflows/build.yml:62]: You are aware that the change you propose won't use the supplied secret `github_repo_actor` and `github_repo_token` when supplied by the project right ?

Comment (kenzhang-ces) [reply .github/workflows/build.yml:62]: Yeah... No need to supply these two variables if we can get it directly from github.

Comment (jsoetanto-ces) [reply .github/workflows/build.yml:62]: If @aconstanti-ces fine with it, should be good

Comment (kenzhang-ces) [reply .github/workflows/build.yml:62]: From my understanding, "github_repo_actor" and "github_repo_token" was not defined... so in your PR https://github.com/coexservices/workflows/pull/20 
you try to use CES_GITHUB_PACKAGES_READ_ACTOR & CES_GITHUB_PACKAGES_READ_TOKEN which is already defined in the secrets. 

My approach is try to use the pre-defined github variables and secrets.

**Pull request closed without merging**